drive/client.py

```diff
@@ -92,30 +92,30 @@ def _serialize_query_value(value):
 class Client:
     """Google Drive client"""
 
-    def __init__(self, credentials_path: Optional[str] = None, *, download_retries_count=5):
-        http = authorize_credentials(credentials_path)
-        self.service = discovery.build('drive', 'v3', http=http)
-        self.download_retries_count = download_retries_count
+    def __init__(self, credentials_path: Optional[str] = None, *, download_retries_count: int = 5) -> None:
```

Comment:
this `: int` should not be necessary since `5` is already an `int`.

Reply:
Argh you’re right; I mistakenly thought that because of IntelliJ’s typing, which does show an error on `download_retries_count='5'` even without `: int`.